composer.json

```diff
@@ -34,6 +35,12 @@
         "phpro/grumphp-shim": "^1.5",
         "phpunit/phpunit": "^9 || ^10"
     },
+    "repositories": {
+        "acquia_global_commands": {
+            "type": "vcs",
+            "url": "git@github.com:vishalkhode1/acquia_global_commands.git"
+        }
+    },
```

Comment:
@vishalkhode1 could we initiate the migration of the 'acquia_global_commands' repository from 'vishalkhode1' to the 'acquia' organization? Additionally, release the development version on packagist, so no need to add this section in composer.json file.

Reply:
Yes, will do that once we all (i.e Acquia CMS team, blt team etc.) are align with this approach.